Libraries/LibJS/Runtime/IndexedProperties.cpp

```diff
@@ -346,6 +346,13 @@ void IndexedProperties::append_all(Object* this_object, const IndexedProperties&
     }
 }
 
+void IndexedProperties::set_array_like_size(size_t new_size)
+{
+    if (m_storage->is_simple_storage() && new_size > SPARSE_ARRAY_THRESHOLD)
```

Comment:
When I search for `SPARSE_ARRAY_THRESHOLD` in the codebase, the mix of `<`, `<=` and `>=` makes it pretty rough to understand if the switch happens at 200 or 201 elements.

Reply:
True :confused:

Reply:
Mixing of zero-based indices and "one-based" sizes does not make it easier, too